### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by zmeers** [zmeers/ggparliament#50]
Hi Rob,

Is this package still being actively maintained? If not, would you be interested in handing the package off or adding co-ownership to the repository?

Thanks,
Zoe

**Comment by RobWHickman** [comment]
yes! I've been meaning to for a while but it keeps slipping my mind. https://stackoverflow.com/questions/39223320/transferring-maintainership-of-an-r-package-on-cran suggests I need to send an email then transfer the repo but will look into it on the weekend

**Comment by RobWHickman** [comment]
looks like just a transfer of the repo is needed but I'll do some more digging tomorrow. For now have requested transfer of the repo to you :+1:
